tika/tika.go

```diff
@@ -185,35 +197,35 @@ func (c *Client) ParseRecursive(ctx context.Context, input io.Reader) ([]string,
 
 // Meta parses the metadata from the given input, returning the metadata and an
 // error. If the error is not nil, the metadata is undefined.
-func (c *Client) Meta(ctx context.Context, input io.Reader) (string, error) {
-	return c.callString(ctx, input, "PUT", "/meta")
+func (c *Client) Meta(ctx context.Context, input io.Reader, header http.Header) (string, error) {
```

Comment:
Looks like this still has the new `header` arg.

Reply:
For sure. I'll fix it up. The issue with making the changes before breakfast this morning  😂